cmd/loader.go

```diff
@@ -61,7 +61,7 @@ func (d *DataLoader) loadCases() error {
 		return err
 	}
 	// Find all the temporal data files
-	files, err := filepath.Glob(filepath.Join(d.dataDir, "covid19_county_*.csv"))
+	files, err := filepath.Glob(filepath.Join(d.dataDir, "covid19_county_????-??-??.csv"))
```

Comment:
I had a bunch of unprocessed covid_county_fips files in the data dir that got pulled in and caused me grief until I realized what was going on...